app/src/main/java/com/conupods/MapsActivity.java

```diff
@@ -241,4 +276,27 @@ public static void setSearchViewOnClickListener(View v, View.OnClickListener lis
             }
         }
     }
+
+    public synchronized void buildGoogleAPIClient() {
+        mGoogleAPIClient = new GoogleApiClient.Builder(this)
+                .addConnectionCallbacks(this)
+                .addOnConnectionFailedListener(this)
+                .addApi(LocationServices.API)
+                .build();
+        mGoogleAPIClient.connect();
+    }
+
+
+    @Override
+    public void onConnected(@Nullable Bundle bundle) { }
```

Comment:
Is this and the onConnectionFailed method empty just because they are needed for an interface?

Reply:
Yes exactly it needs to be implemented because of the interface that MapsActivity is implementing, although i only need one of the 3 methods that needed to be implemented